### ISSUE TITLE
No license

### ISSUE THREAD

**Issue by vicvis** [Ryochan7/DS4Windows#793]
I have users in my organization that would like to use this, but there is no license that would give us permission to do so, other than what is implied by the fact it is offered on a public web site with links to download an instructions, etc.  Can you create a license file, even if it says "Do whatever you want with this, I don't care."  It would provide us with better legal confidence.  Thanks!

**Comment by Ryochan7** [comment]
I am not sure if it has ever been explicitly defined but I always just ran under the assumption that DS4Windows was licensed under the MIT license. Although, as demonstrated many times over, open source licenses don't really mean anything. I will add an MIT license file to the project and define it for the repo.

**Comment by Ryochan7** [comment]
A license file has been added to the root folder and GitHub detected the license properly. DS4Windows will use the MIT license.

https://github.com/Ryochan7/DS4Windows/blob/jay/LICENSE.txt

**Comment by mika-n** [comment]
This is good. 

Originally about year ago when I was looking for a solution about how to use DS4 controller to steer a truck in EuroTruckSimulator2 I run into JayKings/DS4Windows. Then I learned that the JayKing had moved on to concentrate on other things and the DS4Windows app in ds4windows.com web site had not been updated for years.

The next app I run into was InputMapper. However, after reading more in "Internet" I realized that it was actually a sort of fork of JayKings version. Fork or re-write? I dont know but the spiritual father of InputMapper was Jay2King's open sourced version. However, InputMapper had gone as closed source solution. This looked a bit shady to me. Jay2King's source package doesn't have any license files, but I'm sure his intention was that any fork of his great base work should remain open sourced also and a fork should give him the credit he deserves.

Then finally I run into this Ryochan7 fork of DS4Windows app and phew what a relief. This fork didn't try to hide and take away anything from Jay2Kings even when Ryochan7 has done a massive job over the years to keep the app alive and improving it. I too was happy to contribute changes, new features and fixes into this Rychan7/DS4Windows fork because it was not a shady fork of Jay2King's original work, but "legitimate" fork remaining open sourced also as the original version.

This new license file clears up things even more.

(PS. The references to InputMapper and how it may have been fork of Jay2King's original open source version in the past but then went closed source is my own opinion. That's how I see it, but I could be wrong).
